.github/workflows/python_test.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Run Linters
         run: |
           source venv/bin/activate
-          flake8 python
+          flake8 python --ignore=E501
```

Comment:
how about adding this to setup.cfg so it's applied in all scenarios including local development runs?

Reply:
i'd defer that to later until more options are used